src/SDK.ts

```diff
@@ -42,8 +42,9 @@ import { CustomContextData } from "./Utils/CustomContextData";
 
 export default class SDK implements ISDK {
   private static defaultConfiguration: ISDKConfiguration = {
-    getChatTokenRetryCount: 35,
+    getChatTokenRetryCount: 10,
```

Comment:
did we set on not allow the customer to tune this values as he wants  via config json in lcw?

Reply:
Based on investigation, we will not reach `35` tries. We reduce the value as a preemptive measure.

Reply:
This value is abstracted from customer already. LCW has a config to explicitly see this value which is not exposed to customer.